frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableResizableContent/ticketsTableResizableContent.component.tsx

```diff
@@ -35,14 +34,9 @@ export type TicketsTableResizableContentProps = {
 export const TicketsTableResizableContent = ({ setTicketValue, groupBy, selectedTicketId }: TicketsTableResizableContentProps) => {
 	const { template } = useParams<DashboardTicketsParams>();
```

Comment:
I sometimes cant access the cog to set which columns are visible because the resizer is there:

<img width="118" alt="image" src="https://github.com/user-attachments/assets/5e8833e4-2116-441a-ba80-cb58f9ea609a" />